### DISCUSSION TITLE
［求助］xdm，怎么删帖呀，我好像发了俩一样的

### DISCUSSION THREAD

**Discussion by ITwTIb** [share121/inter-knot#1985] [其他]
终于，进来了
![1000021059](https://github.com/user-attachments/assets/448affff-2d48-4e74-8c9f-12891edf3dff)

**Comment by lcshxl** [comment]
可能是点开头像然后在自己发的帖子那里点击三个点？我也不知道

**Comment by Dave-12138** [comment]
![删帖示意图](https://github.com/user-attachments/assets/1bf955b4-9407-4884-b727-295962727142)

点这个垃圾桶
